github4s/src/main/scala/github4s/Decoders.scala

```diff
@@ -278,4 +278,37 @@ object Decoders {
           starred_at <- c.downField("starred_at").as[String]
           user       <- c.downField("user").as[User]
         } yield Stargazer(Some(starred_at), user)))
+
+  implicit val decodeTeam: Decoder[Team] =
```

Comment:
Can the team decoder be automatically derived?

implicit val decoderTeam: Decoder[Team]               = deriveDecoder[Team]